composites/SnippetPreview/components/InputField.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { EditorState, Editor } from "draft-js";
+import colors from "../../../style-guide/colors.json";
+import styled from "styled-components";
+
+const InputFieldContainer = styled.div`
+	cursor: text;
+	height: 16px;
+	border: 1px solid ${ colors.$color_grey };
+	width: 100%;
+	padding: 3px 5px;
+	margin: 5px 1px 1px 1px;
+	background-color: ${ colors.$color_white };
+	overflow: hidden;
+	font-size: 14px;
+	font-style: inherit;
+	.public-DraftStyleDefault-block {
+        white-space: nowrap;
```

Comment:
looks like wrong indentation here, but when I checkout this branch locally and check this part, the indentation is done correctly, not sure why github does this.